examples/classification/convert_from_original/convert_vgg.py

```diff
@@ -0,0 +1,37 @@
+import chainer
```

Comment:
1. I think that it is OK to leave conversion code under the current directory. This is because unlike other examples, VGG does not have its own `demo.py` or `train.py`.
2. OK